static/scss/answers/collapsible-filters/inactive-elements.scss

```diff
@@ -0,0 +1,5 @@
+@media (max-width: #{$breakpoint-collapsed-filters}) {
```

Comment:
before this styling was only being applied to vertical standard/grid, is it okay to have it apply to all page templates?

Reply:
Ya, we definitely want to hide it on all page templates, however after double checking on vertical-map I realized that it's not optimal. This is because vertical map removes the answers-footer at a smaller breakpoint than vertical-grid and vertical-standard remove the answers-footer, so if we leave it as is we will remove the YxtFooter a little bit earlier than necessary on the vertical-map. We currently can't target different page templates for the YxtFooter because that footer is outside the div which contains the css class of the vertical page template. One solution would be to update the page templates to include the page template name on the Yxt-Page-wrapper div. How does that sound?

Reply:
Update: That won't work because Yxt-Page-wrapper is outside of the scope of the page templates so there no straightforward way of specifying that per template without some hacky js. I'm investigating solutions for this through Interactions.js

Reply:
2nd Update: We can put the name of the template on the footer as a css class and target it that way